ui/app/src/components/Header.tsx

```diff
@@ -11,17 +11,99 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { AppBar, Switch, Toolbar, Typography } from '@material-ui/core';
+import {
+  AppBar,
+  Box,
+  Button,
+  CircularProgress,
+  Divider,
+  Menu,
+  MenuItem,
+  Switch,
+  Theme,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import { ChevronDown } from 'mdi-material-ui';
+import { SxProps } from '@mui/system/styleFunctionSx/styleFunctionSx';
+import { MouseEvent, useState } from 'react';
+import { ProjectModel, useProjectQuery } from '../model/project-client';
+import Toast from './Toast';
+
+function ProjectMenu(props: { projectList: ProjectModel[] }): JSX.Element {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleMenu = (event: MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleCloseMenu = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <>
+      <Button
+        aria-label="list of the available project"
+        aria-controls="menu-project-list-appbar"
+        aria-haspopup="true"
+        color="inherit"
+        endIcon={<ChevronDown />}
+        onClick={handleMenu}
+      >
+        Projects
+      </Button>
+      <Menu
+        id="menu-project-list-appbar"
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+        keepMounted
+        open={Boolean(anchorEl)}
```

Comment:
This may be clearer / more explicit:

```suggestion
        open={anchorEl !== null}
```